perun-core/src/main/java/cz/metacentrum/perun/core/implApi/TasksManagerImplApi.java

```diff
@@ -26,6 +26,8 @@
 	 */
 	List<Task> listAllTasksForFacility(int facilityId);
 
+	List<Task> listAllTasksForService(int serviceId);
```

Comment:
Please, add also a javadoc for this method.

Reply:
Added.